src/models/event.ts

```diff
@@ -1,5 +1,15 @@
+import { type } from "os"
+
 export interface Event {
+
     start: Date
     end: Date
     title: string
+    color: string
+    modalTitle: string
+    modalDescription: string
+    type: 'checkin' | 'booking'
```

Comment:
@GabrielDVpereira Do you remember how to do so the local and the status depends on the type? As we've seen the other day?